synapse/rest/client/room.py

```diff
@@ -916,6 +916,9 @@ def __init__(self, hs: "HomeServer"):
         self.room_member_handler = hs.get_room_member_handler()
         self.auth = hs.get_auth()
 
+    def _has_3pid_invite_keys(self, content: JsonDict) -> bool:
+        return all(key in content for key in ("medium", "address"))
```

Comment:
since you're rewriting this anyway: I think you may as well inline it. Having a separate function just makes it harder to read.